### DISCUSSION TITLE
Utilities ideas for the next version (1.0.13)

### DISCUSSION THREAD

**Discussion by fatmanawfal** [EnabApp/nuxt3#97] [Ideas]
**Hi everyone**
the purpose of our application is to make it more usability and flexibility for business management, so I want to discuss with you some of ideas.
**### CALC. APP**
What about add more arithmetic operations such as (^2, ^3, roots, ...etc.)? most of businessmen are not limited their work only on (+, -, ...). 

**### TODO APP**
Add (sub-task) in every single task.

also the apps need more flexibility in size, you should try when you maximize or minimize app size it doesn't respond to the size you want especially maximize size **it lacks flexibility.**

so what is your opinion about this ideas and how we can improve our apps?

---

- [x]  create composable to add task

**Comment by Gruce** [comment]
Hello @fatmanawfal,

**1. Calculator:**
Yes could have (^2 and ^3 and roots) and I think that is enough where the businessmen not really dealing with the numbers themselves in this deep. But they have accountants may benefit this features.

**2. Todo:**
Yes, I think @a7med3bd could help with these sizes lacks.

Thank you.

**Comment by i1Zeus** [comment]
>**### CALC. APP** What about add more arithmetic operations such as (^2, ^3, roots, ...etc.)? most of businessmen are not limited their work only on (+, -, ...).
> 
> **### TODO APP** Add (sub-task) in every single task.
> 
> also the apps need more flexibility in size, you should try when you maximize or minimize app size it doesn't respond to the size you want especially maximize size **it lacks flexibility.**
> 
> so what is your opinion about this ideas and how we can improve our apps?

Hey @fatmanawfal , I have already implemented these features in calc app... you can check the new release.
about the todo app I don't think I understood u well... why we need a sup task in task? the user can add multiple task and use it as regular.

**Reply by fatmanawfal** [reply]
let's imagine you have two main tasks for today, 
1- planning for new project
2- complete when you stop in current project
in every single task you have different task to do and you need to put more to do tasks but related to the main task
if you don't understand I can share with you screen in the discord to show you the benefit of sub-task

**Reply by i1Zeus** [reply]
**Gotcha!** But I don't thing that is functional the way u see it... we're trying to make our apps as simp as they can be, for me adding multiple task is easier then adding a task in another one!

**Comment by Gruce** [comment]
Hello @fatmanawfal & @i1Zeus I think we need composable to add todo from external app.


Thank you

**Reply by fatmanawfal** [reply]
Hi,
can you tell us more details what did you mean in external app?

**Reply by Gruce** [reply]
Anyone can add task from it's app. For example from Supermarket app I can add task using function (composable) your provide.

**Reply by fatmanawfal** [reply]
OK, I get it
